### ISSUE TITLE
Accurate problem

### ISSUE THREAD

**Issue by sssserious** [aleflabo/MoCoDAD#24]
Thank you for your open source project. I used Readme to train the Mocodad model, and then Eval_mocodad found that on the Ubnormal dataset, the accuracy rate was about 64-65%. It was much different from the best performance in the paper. I trained 4 times in the default parameter, each time it was very different from the best performance. I used visual data and found that the best AUC on the Ubnormal verification set was about 75%, and the initial code PL only saved the TOP2 CKPT. Is there any problem with my experiments? Can you give specific experimental details or the specific parameter details on the UBNORMAL verification set?

**Comment by stdrr** [comment]
Dear @sssserious, thank you for your interest in our project!
 
While the provided checkpoints yield the results we reported in the paper, we could not recover the right hyperparameters to reproduce the best checkpoint from scratch. The publicly released hyperparameters allow us to match our results more closely, but some more tuning may be required; unfortunately, we don't have much time at the moment to carry out some more experiments. 

I apologize for the inconvenience and hope our work can be helpful with your research anyway.

Best regards,
Stefano
